src/control/cmd/daos/container.go

```diff
@@ -1095,7 +1095,6 @@ func (cmd *containerCloneCmd) Execute(_ []string) error {
 	}
 
 	// Compat with old-style output
```

Comment:
Does this comment want to go as well?

Reply:
I'm not sure. The other message below is also "old-style"
`cmd.Infof("Successfully copied to destination container %s", C.GoString(&ap.dm_args.dst_cont[0]))`